src/main/java/library/app/spring/entity/Author.java

```diff
@@ -25,7 +27,10 @@
     @Column(name = "surname")
     private String surname;
 
-    @ManyToMany(mappedBy = "authors", fetch = FetchType.EAGER)
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
why we use `join table` in author and book models?

Reply:
change on mapped